### ISSUE TITLE
[1.7.10] Ex-Utils Energy Generation Bug

### ISSUE THREAD

**Issue by radixwolf** [aidancbrady/Mekanism#4210]
# tl;dr universal cables give infinite energy to ex-utils generators when they're not producing energy

When playing in my multiplayer server with a friend, we were confused on why our 64x Lava generator was not capping, it was around 20,000,000 of the 6,000,000 cap, we found out that any and all Mekanism universal cables will hold the energy sent through them, but when they aren't producing energy at all, the cable will give off its energy it was holding to the generator 

_(normally generators aren't supposed to take **IN** power, but in this case it does for some reason)_ 

Having this happen produces an infinite amount of energy to the generator, along with an infinite limit of power in the generator itself, overriding the cap of the generator.

_(I did test this with energy conduits from ex-utils and the issue doesn't happen, meaning this is a mekanism issue)_

[Video of an example](https://youtu.be/QvYu7LloL38)

**[Current Mod Pack]**
Project Ozone 2 RELOADED 2.2.2

**[Current Mod Versions]**
Mekanism-1.7.10-9.1.0.281
extrautilities-1.2.12

**Comment by ThatGamerBlue** [comment]
1.7.10 isn't supported. Sorry!

**Comment by radixwolf** [comment]
> 1.7.10 isn't supported
what do you mean? thats the version this mod plays on

**Comment by ThatGamerBlue** [comment]
1.10.2 is the latest version.
